core/src/main/java/io/lionweb/lioncore/java/serialization/DeserializationStatus.java

```diff
@@ -92,14 +99,19 @@ Node resolve(@Nullable String nodeID) {
   @Nonnull
   ProxyNode createProxy(@Nonnull String nodeID) {
     Objects.requireNonNull(nodeID, "nodeID should not be null");
-    ProxyNode proxyNode = this.serialization.createProxy(nodeID);
+    if (complexiveInstanceResolver.resolve(nodeID) != null) {
+      throw new IllegalStateException(
+          "Cannot create a Proxy for node ID "
+              + nodeID
+              + " has there is already a Classifier Instance available for such ID");
```

Comment:
typo: "as"
Might be helpful to mention the existing classifier instance

Reply:
Corrected, thank you